flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/PulsarSourceOptions.java

```diff
@@ -124,6 +124,8 @@ private PulsarSourceOptions() {
                                             " We would automatically commit the cursor using the given period (in ms).")
                                     .build());
 
+    /** @deprecated We no longer need transitions for consuming messages. */
```

Comment:
```suggestion
    /** @deprecated We no longer need transactions for consuming messages. */
```

?

Also, if this option has no effect, we'd better directly remove it.